sphinx/config.py

```diff
@@ -349,6 +350,19 @@ def check_unicode(self, warn):
                      'characters; this can lead to Unicode errors occurring. '
                      'Please use Unicode strings, e.g. %r.' % (name, u'Content'))
 
+    def check_extension_blacklist(self, warn):
```

Comment:
I think this checker should move to `sphinx.application.Sphinx.setup_extension()`.
Sphinx allows to call `setup_extension()` to load an extension if the 3rd party extension depends on another extensions.

>Load the extension given by the module name. Use this if your extension needs the features provided by another extension.

http://www.sphinx-doc.org/en/stable/extdev/appapi.html#sphinx.application.Sphinx.setup_extension